pubsub/message.go

```diff
@@ -112,12 +147,37 @@ func (ah *psAckHandler) OnNack() {
 	ah.done(false)
 }
 
+func (ah *psAckHandler) OnAckWithResult() *AckResult {
+	if !ah.exactlyOnceDelivery {
+		return newSuccessAckResult()
+	}
+	ah.ackResult = ipubsub.NewAckResult()
+	ah.done(true)
+	return ah.ackResult
+}
+
+func (ah *psAckHandler) OnNackWithResult() *AckResult {
+	if !ah.exactlyOnceDelivery {
+		return newSuccessAckResult()
+	}
+	ah.ackResult = ipubsub.NewAckResult()
+	ah.done(false)
```

Comment:
Does this `false` mean "done with nack? Perhaps add an inline parameter comment to clarify?

Reply:
Yeah this was inherited from `OnAck()` and `OnNack()`, which shows up slightly before these lines in the same file. I added comments for clarity anyway since these methods are more complicated than the aforementioned ones.